bigquery/spec/data/test_data.csv

```diff
@@ -0,0 +1,3 @@
+"Brent Shaffer","Master of the Deprecated"
```

Comment:
nit: I'd prefer to see generic names used and not see our names in the code anywhere.  eg. Alice and Bob